### ISSUE TITLE
Book Series

### ISSUE THREAD

**Issue by smilerz** [openaudible/openaudible#75]
It would be very useful to also scrape information about the Series that a book is a part of (a link to the series is visible in the library as 'View Series')

Knowing the series order and any unpurchased books in the series are specifically the features I'm looking for.

**Comment by TomW1605** [comment]
TL;DR: Audibal makes retrieving this info both hard and unreliable. for sorting purposes a manually entered field would be a lot easier to implement and not that much trouble for the user..

i have looked into implementing this in my fork of this repo and it is harder than it looks (in Aus anyway). to start with the link does not contain the name of the series and is instead more like a search, eg https://www.audible.com.au/series?asin=B07MBJK7P1, this would mean loading another whole page and extracting the series name and book number from that. second the link is not reliable if a book is in more than one series or the series contains 'mini series', a good example is Terry Pratchett's Discworld series where some books are grouped into a 'mini series' because they have the same main characters and format, because of this the 'View Series' link will not always take you to the full Discworld series. and third even when you can get to the page for the right series it sometimes contains extra books like reference books or books that are actually from the story and not actually part of the series. all of this makes it hard to reliably retrieve the series directly from Audable. however if simply for sorting purposes it would not be hard to implement a manually entered 'series' and 'book number' field that you could then sort by (this is my current plan)

**Comment by openaudible** [comment]
Out of scope.
